plasmapy/dispersion/numerical/hollweg_.py

```diff
@@ -310,12 +308,14 @@ def hollweg(
     c0 = -alpha_s * sigma ** 2
 
     # Find roots to polynomial
-    coefficients = np.array([c3, c2, c1, c0], ndmin=2)
+    coefficients = np.array([c3, c2, c1, c0], ndmin=3)
     nroots = coefficients.shape[0] - 1  # 3
     nks = coefficients.shape[1]
-    roots = np.empty((nroots, nks), dtype=np.complex128)
+    nthetas = coefficients.shape[2]
+    roots = np.empty((nroots, nks, nthetas), dtype=np.complex128)
     for ii in range(nks):
-        roots[:, ii] = np.roots(coefficients[:, ii])[:]
+        for jj in range(nthetas):
+            roots[:, ii, jj] = np.roots(coefficients[:, ii, jj])[:]
```

Comment:
Do we actually need the `[:]` at the end?

Reply:
I see no difference in the output when the `[:]` is removed. Do you remember why this was added @rocco8773 ?

Reply:
```suggestion
            roots[:, ii, jj] = np.roots(coefficients[:, ii, jj])
```

Yes, the `[:]` at the end is not required in this case.